jbuilder.opam

```diff
@@ -6,17 +6,9 @@ bug-reports: "https://github.com/ocaml/dune/issues"
 dev-repo: "https://github.com/ocaml/dune.git"
 license: "Apache-2.0"
 build: [
+  ["ocaml" "configure.ml" "--library-path" lib]
```

Comment:
This is a typo? You mean libdir perhaps.

Reply:
Indeed, fixed